shared.js

```diff
@@ -13,7 +13,7 @@ export function SetOptions(items) {
 	options = items;
 }
 
-function unblockUser(user, user_id, headers, reason) {
+function UnblockUser(user, user_id, headers, reason, attempt = 1) {
```

Comment:
- Renamed for consistency with the rest of the file. 
- `attempt` was being used but never defined, like in `BlockUser()`

Reply:
the idea behind the naming convention is going more toward's golang's styling of having exported objects be PascalCase and non-exported objects being camelCase. obviously I don't follow that very closely, but that was the idea. good catch on the attempt missing!

Reply:
I kind of thought that, but saw other non-exported functions. I think this was the only function in this file that was camelCase.